client/web/web.go

```diff
@@ -66,6 +66,11 @@ const (
 	sessionCookieExpiry = time.Hour * 24 * 30 // 30 days
 )
 
+var (
+	exitNodeRouteV4 = netip.MustParsePrefix("0.0.0.0/0")
+	exitNodeRouteV6 = netip.MustParsePrefix("::/0")
+)
```

Comment:
I wonder if we actually have these vars somewhere in a lib... But this is a nice update!

Reply:
we seem to have them at least 3 times in this repo.  maybe the copy in `net/netutil` would be the best to export and reuse?

Reply:
maybe in a separate PR though, since @bradfitz might have opinions on exporting those long-term?